pkg/kubelet/eviction/threshold_notifier_linux.go

```diff
@@ -18,42 +18,44 @@ package eviction
 
 import (
 	"fmt"
+	"sync"
+	"time"
 
 	"github.com/golang/glog"
 	"golang.org/x/sys/unix"
 )
 
-type memcgThresholdNotifier struct {
-	watchfd     int
-	controlfd   int
-	eventfd     int
-	handler     thresholdNotifierHandlerFunc
-	description string
+const (
+	// eventfdSize is the number of bytes returned by a successful read from an eventfd
+	// see http://man7.org/linux/man-pages/man2/eventfd.2.html for more information
+	eventfdSize = 8
```

Comment:
nit: it's the size of the event, not the size of the fd, right?

Reply:
done